README.md

```diff
@@ -51,6 +51,42 @@ This document provides the information needed to create a DASH packager that is
 
   See https://github.com/google/edash-packager/blob/master/CONTRIBUTING.md for details.
 
+# Using docker for testing/development #
+
+1. Requirements:
+
+   1. Clone this repo.
```

Comment:
Great :+1: should I only update README? and discard Dockerfile, like pointing to my git?

or the point was to remove `clone this repo` from instructions? and still point to edash official repo?

```shell
docker build -t edash github.com/google/edash-packager.git
```

ps: I'll correct README later.